src/api_crypto.rs

```diff
@@ -524,3 +526,44 @@ impl SequesterVerifyKeyDer {
 pub(crate) fn generate_nonce(py: Python<'_>) -> &PyBytes {
     PyBytes::new(py, &libparsec::crypto::generate_nonce())
 }
+
+#[pyclass]
+#[derive(Clone)]
+pub(crate) struct Share(libparsec::crypto::Share);
+
+#[pymethods]
+impl Share {
+    #[classmethod]
+    fn load(_cls: &PyType, raw: &[u8]) -> PyResult<Self> {
+        libparsec::crypto::Share::try_from(raw)
+            .map(Self)
+            .map_err(PyValueError::new_err)
+    }
+
+    fn dump<'py>(&self, py: Python<'py>) -> &'py PyBytes {
+        PyBytes::new(py, &Vec::from(&self.0))
+    }
+}
+
+#[pyclass]
+pub(crate) struct Sharks(libparsec::crypto::Sharks);
```

Comment:
Why do we choose the name `Sharks` ?

Reply:
To keep the same name between Rust/Python, I didn't wrap Sharks, I reexposed it

Reply:
Still I don't find the name `Sharks` self explanatory, we could rename it

Reply:
It's no longer called `sharks`